### PR TITLE
div. Features

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: grails-app/i18n/messages_de.properties [hunk 1]
```diff
@@ -4918,6 +4918,7 @@ openSurvey.button =Umfrage nur freigeben
 openSurvey.button.info =Die Umfrage können Sie erst freigeben, wenn Sie Umfrage-Mermale festgelegt und Teilnehmer verknüpft haben.
 openSurvey.button.info2 = Die Umfrage wird erst gestartet, wenn das Startdatum erreicht ist.
 openSurvey.successfully =Die Umfrage wird automatisch freigegeben, wenn das Start-Datum erreicht ist.
+openSurvey.openFailByTitleSelection.noPackagesYetAdded = Sie haben noch kein Paket mit den Teilnehmer-Lizenzen verknüpft. Bitte die Teilnehmer-Lizenzen �berpr�fen.
```
Comment (klodav) [inline grails-app/i18n/messages_de.properties:4921]: Encoding

File: grails-app/i18n/messages_de.properties [hunk 2]
```diff
@@ -5079,6 +5080,7 @@ renewEntitlementsWithSurvey.currentTitles.button = Zu den Bestandstiteln
 renewEntitlementsWithSurvey.currentTitlesSelect.button = Zu den ausgewählten Titel
 renewEntitlementsWithSurvey.stats.button = Zur Statistik der Gesamttitelliste
 renewEntitlementsWithSurvey.transferPerpetualAccessTitles = Alte dauerhafte Titel übertragen
+renewEntitlementsWithSurvey.noPackagesYetAdded = Sie haben noch kein Paket mit der Lizenz des Teilnehmers verknüpft. Bitte ein Paket verknüpft um  die ausgewählte Titel zu speichern.
```
Comment (klodav) [inline grails-app/i18n/messages_de.properties:5083]: Encoding

File: grails-app/i18n/messages_de.properties [hunk 3]
```diff
@@ -4918,6 +4918,7 @@ openSurvey.button =Umfrage nur freigeben
 openSurvey.button.info =Die Umfrage können Sie erst freigeben, wenn Sie Umfrage-Mermale festgelegt und Teilnehmer verknüpft haben.
```
Comment (klodav) [inline grails-app/i18n/messages_de.properties:4918]: Encoding

File: grails-app/i18n/messages_de.properties [hunk 4]
```diff
@@ -4918,6 +4918,7 @@ openSurvey.button =Umfrage nur freigeben
...
 openSurvey.button.info2 = Die Umfrage wird erst gestartet, wenn das Startdatum erreicht ist.
 openSurvey.successfully =Die Umfrage wird automatisch freigegeben, wenn das Start-Datum erreicht ist.
+openSurvey.openFailByTitleSelection.noPackagesYetAdded = Sie haben noch kein Paket mit den Teilnehmer-Lizenzen verknüpft. Bitte die Teilnehmer-Lizenzen �berpr�fen.
 
 openSurveyNow.button =Umfrage jetzt direkt starten
 openSurveyNow.button.info =Die Umfrage können Sie erst starten, wenn Sie Umfrage-Mermale festgelegt und Teilnehmer verknüpft haben.
```
Comment (klodav) [inline grails-app/i18n/messages_de.properties:4924]: Encoding

File: grails-app/i18n/messages_de.properties [hunk 5]
```diff
@@ -5079,6 +5080,7 @@ renewEntitlementsWithSurvey.currentTitles.button = Zu den Bestandstiteln
 renewEntitlementsWithSurvey.currentTitlesSelect.button = Zu den ausgewählten Titel
```
Comment (klodav) [inline grails-app/i18n/messages_de.properties:5080]: Encoding

File: grails-app/i18n/messages_de.properties [hunk 6]
```diff
@@ -5079,6 +5080,7 @@ renewEntitlementsWithSurvey.currentTitles.button = Zu den Bestandstiteln
...
 renewEntitlementsWithSurvey.stats.button = Zur Statistik der Gesamttitelliste
 renewEntitlementsWithSurvey.transferPerpetualAccessTitles = Alte dauerhafte Titel übertragen
```
Comment (klodav) [inline grails-app/i18n/messages_de.properties:5082]: Encoding

File: grails-app/i18n/messages_de.properties [hunk 7]
```diff
@@ -5079,6 +5080,7 @@ renewEntitlementsWithSurvey.currentTitles.button = Zu den Bestandstiteln
...
+renewEntitlementsWithSurvey.noPackagesYetAdded = Sie haben noch kein Paket mit der Lizenz des Teilnehmers verknüpft. Bitte ein Paket verknüpft um  die ausgewählte Titel zu speichern.
 
 renewEntitlementsWithSurvey.issueEntitlementSelect.label = Titel-Vorauswahl
 renewEntitlementsWithSurvey.issueEntitlementSelect.info = Upload der Excel-Gesamttitelliste mit den von Ihnen ausgewählten Titeln für eine automatische Mehrfachauswahl
```
Comment (klodav) [inline grails-app/i18n/messages_de.properties:5086]: Encoding

**Review by klodav — CHANGES_REQUESTED** [review]
todo

**Comment by Moetez16** [comment]
PR geupdatet.

**Review by klodav — APPROVED** [review]

**Pull request merged**